test/expectations/inline-source-map.js

```diff
@@ -0,0 +1,11 @@
+"format register";
+
+(function() {
+function define(){};  define.amd = {};
+System.register("tree/amd-2", [], false, function() {
+  return {amd: '2'};
+});
+
+
+})();
+//# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbInRyZWUvYW1kLTIuanMiXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6Ijs7Ozs7T0FPTyxFQUFFLEdBQUUsQ0FBRyxJQUFFLENBQUU7RUFBRTtBQUFBIn0=
```

Comment:
Could replace this full-text match with a few assertions and keep high confidence:

1. Only one `//# sourceMappingURL` comment, and it's on the last line
2. Type is `data:application/json;base64`
3. Decoding and parsing it generates a parseable source map